packages/core/package.json

```diff
@@ -31,7 +31,8 @@
   },
   "dependencies": {
     "@module-federation/runtime": "^0.1.2",
-    "@starknet-io/types-js": "^0.7.7"
+    "@starknet-io/types-js": "^0.7.7",
```

Comment:
shouldn't it be under `devDependencies`?

Reply:
Hi , u mean the `"@starknet-io/types-js` ? didnt aware it is in dependencies
it looks like this is an existing issue, 
i would suggest create another PR to fix it